src/components/Statistic/Statistic.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+class Statistic extends React.Component {
```

Comment:
No state being used inside this component so we could use a stateless component

Reply:
on it